corehq/apps/app_manager/views/forms.py

```diff
@@ -260,14 +260,15 @@ def should_edit(attribute):
 
     if should_edit("name"):
         name = request.POST['name']
-        form.name[lang] = name
+        form.name[lang] = html_name(name)
```

Comment:
I'm a little surprised it's supported to put HTML in a form name. Also don't understand why we want to put `<span class="label label-info">Empty</span>` in the name if it's empty.